src/components/Calculator.js

```diff
@@ -1,37 +1,78 @@
 import '../scss/style.scss';
 import React from 'react';
+import calculate from '../logic/calculate';
 
```

Comment:
- [ ] The Calculator component should import the `calculate.js & operate.js` files from the logic directory but currently, you are importing `calculate` only please fix that

Reply:
@DeliceLydia operate is already imported and used by calculate.js, hence I do not have any use for it in Calculator.js. It throws an error when imported as I will not need to call it.